spec/moleculer/service/base_spec.rb

```diff
@@ -1,7 +1,10 @@
+# frozen_string_literal: true
+
 RSpec.describe Moleculer::Service::Base do
   subject do
     Class.new(Moleculer::Service::Base) do
       service_name "some-service"
+      version "beta-0.0.1"
```

Comment:
We should also have a test for the Numeric version since the full_name method has specific logic around that condition.